src/obiwan/core.clj

```diff
@@ -219,9 +219,7 @@
   (.zrange redis k zmin zmax))
 
 (defn zrange-with-scores [^UnifiedJedis redis k zmin zmax]
-  (->> (.zrangeWithScores redis k zmin zmax)
-       (map #(c/<-tuple %))
-       (into {})))
+  (.zrangeWithScores redis k zmin zmax)) ; set with scores are returned as tuples, and if needed can be converted using obiwan.commands/<-tuple.
```

Comment:
^^ only to make the obi wan-library clean, as pipeline query would fail otherwise with below error :
<img width="735" alt="image" src="https://github.com/tolitius/obiwan/assets/37661954/6cfed988-b138-48b2-be6a-ef76c4481dc8">